### PR TITLE
clean up deployment scripts

### PR DESCRIPTION
**Pull request by jellegerbrandy**

### CODE REVIEW

File: deployEthereum.js [hunk 1]
```diff
@@ -4,19 +4,19 @@ const path = require('path')
...
   output: path.normalize(path.join(__dirname, './migration.json')),
   params: JSON.parse(fs.readFileSync(path.join(__dirname, 'migration-params.json')))
 }
 
 void async function() {
   const DAOstackMigration = require('@daostack/migration');
   const setupTestEnv = require('./setup-test-env')
-
-  DAOstackMigration.migrateScript(setupTestEnv)(options)
+  // await DAOstackMigration.migrateBase(options)
```
Comment (ben-kaufman) [inline deployEthereum.js]: remove commented line

Comment (ben-kaufman) [reply deployEthereum.js]: Also other commented lines from above

File: package.json [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
   "main": "index.js",
   "scripts": {
-    "migrate": "node ./migrate.js",
     "release": "./release.sh",
-    "prepare-subgraph": "npm explore @daostack/subgraph -- npm i && node setup-subgraph.js",
-    "deploy-subgraph": "npm run prepare-subgraph && npm explore @daostack/subgraph -- npm run deploy",
-    "reset-migration": "cp -f node_modules/@daostack/migration/migration.json ./migration.json"
+    "deploy-ethereum": "node ./deployEthereum.js",
+    "deploy-subgraph": "node ./deploySubgraph.js",
+    "xxxreset-migration": "cp -f node_modules/@daostack/migration/migration.json ./migration.json"
```
Comment (ben-kaufman) [inline package.json]: What's that for?

File: setup-test-env.js [hunk 3]
```diff
@@ -13,9 +13,9 @@ async function assignGlobalVariables (web3, spinner, opts, logTx, previousMigrat
 }
 
 async function migrateDemoTest ({ web3, spinner, confirm, opts, migrationParams, logTx, previousMigration }) {
-  if (!(await confirm('About to migrate new Demo Test. Continue?'))) {
-    return
-  }
+  // if (!(await confirm('About to migrate new Demo Test. Continue?'))) {
```
Comment (ben-kaufman) [inline setup-test-env.js]: Remove...

File: package.json (hunk 2 above)
Comment (jellegerbrandy) [reply package.json]: you wrote it, I just added the xxx's to mark is as "why would you want  to do that?" 
removed it :-)

File: deployEthereum.js (hunk 1 above)
Comment (jellegerbrandy) [reply deployEthereum.js]: no I'd like to keep this for reference. A problem with the current setup building on top of the ganache image from the mgiration repo makes it very hard to upgrade the ganache version;d build from an empty ganache image and uncomment this line :-)

**Comment by ben-kaufman** [comment]
Please resolve the merging conflicts in the code

**Comment by jellegerbrandy** [comment]
This PR does not work as-is because it depends on https://github.com/daostack/subgraph/pull/332

Also, the migrate.js script does not play well with version 24, and needs to e fixed

**Pull request merged**